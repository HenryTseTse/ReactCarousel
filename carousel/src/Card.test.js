import { render, fireEvent } from "@testing-library/react";
import Card from "./Card";

it("should render", () => {
    render(<Card />);
});

it("should match snapshot", () => {
    const {asFragment} = render(
        <Card
            caption="This is a Shiba Inu"
            src="https://dogtime.com/wp-content/uploads/sites/12/2011/01/GettyImages-653001154-e1691965000531.jpg?w=1024"
            currNum={4}
            totalNum={4}
        />
    );
    expect(asFragment()).toMatchSnapshot();
});