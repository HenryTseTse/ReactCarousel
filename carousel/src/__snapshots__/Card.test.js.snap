// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      This is a Shiba Inu
    </h4>
    <img
      alt="This is a Shiba Inu"
      class="Card-image"
      src="https://dogtime.com/wp-content/uploads/sites/12/2011/01/GettyImages-653001154-e1691965000531.jpg?w=1024"
    />
    <small
      class="Card-small"
    >
      Image 4 of 4.
    </small>
  </div>
</DocumentFragment>
`;
